### PR TITLE
docs: update troubleshooting.md with /opt path

### PR DESCRIPTION
**Pull request by DavSanchez**

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The Infrastructure agent installs bundled binaries together in the following dir
 
 ```shell
 # Linux
-/var/db/newrelic-infra/newrelic-integrations/bin/
+/opt/newrelic-infra/newrelic-integrations/bin/
```
Comment (rubenruizdegauna) [inline docs/troubleshooting.md:9]: Maybe it would be good to add both paths and versions (>=1.48.0)

Comment (DavSanchez) [reply docs/troubleshooting.md:9]: hmm yes, fixing!

**Review by rubenruizdegauna — APPROVED** [review]
LGTM

File: docs/troubleshooting.md (hunk 1 above)
Comment (DavSanchez) [reply docs/troubleshooting.md]: Done in https://github.com/newrelic/nri-flex/pull/454/commits/d16bdf445f61bb29ba46dae6a4ed87a741ced539!

**Pull request merged**